src/shared/infrastructure/profile-metrics/profile-metrics.json

```diff
@@ -113,28 +113,28 @@
   },
   "process_cpu:lock_count:count:cpu:nanoseconds": {
     "id": "process_cpu:lock_count:count:cpu:nanoseconds",
-    "description": "Count of lock acquisitions attempted during CPU time",
+    "description": "Number of lock acquisitions attempted during CPU time",
     "type": "lock_count",
     "group": "locks",
     "unit": "short"
   },
   "process_cpu:lock_time:nanoseconds:cpu:nanoseconds": {
     "id": "process_cpu:lock_time:nanoseconds:cpu:nanoseconds",
-    "description": "Cumulative time spent acquiring locks, in nanoseconds",
+    "description": "Cumulative time spent acquiring locks",
     "type": "lock_time",
     "group": "locks",
     "unit": "ns"
   },
   "process_cpu:samples:count::milliseconds": {
     "id": "process_cpu:samples:count::milliseconds",
-    "description": "Total number of process samples collected",
+    "description": "Number of process samples collected",
     "type": "samples",
     "group": "process_cpu",
     "unit": "short"
   },
   "process_cpu:samples:count:cpu:nanoseconds": {
     "id": "process_cpu:samples:count:cpu:nanoseconds",
-    "description": "Count of samples collected over CPU time",
+    "description": "Number of samples collected over CPU time",
```

Comment:
Same question about nanoseconds here. It would be something to add, even though it wasn't present before.

Reply:
In this case, it's a short number (see below in the JSON: `"unit": "short"`),